utils.js

```diff
@@ -1,14 +1,139 @@
 const format = require("pg-format")
 const db = require("./db/connection")
 
-exports.checkIfIdExists = (table,column,id)=>{
-    const query = format(` SELECT * FROM %I where %I = $1;`, table,column)
+/*
 
-    return db.query(query,[id])
-    .then(({rows}) =>{
-        if(!rows.length){
-            Promise.reject({status:404,msg:"not found"})
-        }
-        // return rows
-    })
-}
\ No newline at end of file
+[
```

Comment:
try not to commit big chunks of commented out code like this

Reply:
i was trying something and forgot about it when i didn't need it anymore, will change 👍